### PR TITLE
[wip] Popper

### PR DESCRIPTION
**Pull request by ozerovs**

### CODE REVIEW

**Review by suprMax — COMMENTED** [review]
Could we port these into @goodhood/components? Since we're rewriting them. @peterhass  thoughts?

**Comment by peterhass** [comment]
> Could we port these into @goodhood/components? Since we're rewriting them. @peterhass thoughts?

Let's port them to @goodhood/components. Don't see any reason why not

File: src/feature_alert/tooltip.es [hunk 1]
```diff
@@ -98,6 +110,12 @@ FeatureAlertTooltip.propTypes = {
     POSITION_LEFT,
     POSITION_RIGHT,
   ]),
+  fallbackPosition: PropTypes.oneOf([
+    POSITION_TOP,
+    POSITION_BOTTOM,
+    POSITION_LEFT,
+    POSITION_RIGHT,
+  ]),
```
Comment (peterhass) [inline src/feature_alert/tooltip.es:118]: where does the prop get used?

File: src/tooltip/index.es [hunk 2]
```diff
@@ -1,21 +1,54 @@
...
+
+  const handleOpen = useCallback((event) => {
+    if (event) event.stopPropagation();
+    if (isOpen) return;
+    setOpen(true);
+  }, [isOpen]);
+
+  const handleClose = useCallback(() => {
+    if (!isOpen) return;
+    setOpen(false);
```
Comment (peterhass) [inline src/tooltip/index.es:28]: do we really need the `isOpen` check? Not sure if setting the same value again would trigger a render

`setOpen(false)` -> render
`setOpen(false)` -> does not render?

**Comment by peterhass** [comment]
Would it make sense to extract this popper stuff and create some generic components ourself? That way we would have less code duplication and it may simplify things.

`<GenericTooltip getArrow={getArrow}>`

**Pull request merged**